loleaflet/js/global.js

```diff
@@ -555,7 +555,7 @@
 				// Unless we are backed up.
 				if (that.msgInflight <= 3)
 				{
-//					console.debug('Have data to send, lowering poll interval.');
+					//					console.debug('Have data to send, lowering poll interval.');
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.